websites/R/Rate Your Music/presence.ts

```diff
@@ -0,0 +1,286 @@
+const enum Assets {
+	Logo = "https://i.imgur.com/4hrNGBS.png",
+}
+
+let browsingTimestamp: number;
+
+function updatePresence() {
+	if (!browsingTimestamp) browsingTimestamp = Math.floor(Date.now() / 1000);
+
+	let presenceData: PresenceData = {
+		largeImageKey: Assets.Logo,
+		startTimestamp: browsingTimestamp,
+	};
+
+	const pages: Record<string, PresenceData> = {
+			"/charts/": {
+				details: "Viewing Chart:",
+			},
+			"/new-music/": {
+				details: "New Releases",
+			},
+			"/genres/": {
+				details: "Genres",
+			},
+			"/lists/": {
+				details: "Lists",
+			},
+			"/subscribe/": {
+				details: "Subscribe",
+			},
+			"/": {
+				details: "Home",
+			},
+		},
+		{ pathname } = document.location;
+
+	//chart page
```

Comment:
These comments are mostly redundant and relatively easy to tell from the if statements